src/core_atmosphere/dynamics/mpas_atm_boundaries.F

```diff
@@ -249,6 +321,17 @@ subroutine mpas_atm_update_bdy_tend(clock, streamManager, block, firstCall, ierr
 
         end if
 
+        MPAS_ACC_TIMER_START('mpas_atm_update_bdy_tend [ACC_data_xfer]')
+        if (.not. firstCall) then
+           !$acc exit data copyout(lbc_tend_u, lbc_tend_ru, lbc_tend_rho_edge, lbc_tend_w, &
```

Comment:
Could we add one more space of indentation here to match the indentation in the two lines that follow?

Reply:
Done. thanks for the catch!